DigitalExcellence/dex-backend
# Fix line ending and move from CRLF to LF

## Description

Updated all line endings and started enforcing it using the .gitattributes file

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Open a random file
2. Check if the line endings are LF instead of CRLF

## Link to issue

Closes: #163
